frontend/src/components/common/Card/Card.jsx

```diff
@@ -31,7 +31,7 @@ const Card = (props) => {
   };
 
   return (
-    <div className={styles.card}>
+    <li className={styles.card}>
```

Comment:
For me, it doesn't look good when the component starts with tag `li`. In my opinion, it would be better to leave `div`

Reply:
Why?

Reply:
Because it is a separate component. In my point of view, it should be self-sufficient.